hic_qc.py

```diff
@@ -239,6 +242,8 @@ def parse_bam(self, bamfile, max_read_pairs=-1):
         with pysam.AlignmentFile(self.paths['bamfile']) as bam_fh:
             self.extract_header_info(bam_fh.header)
 
+            print(bam_fh)
```

Comment:
try `self.logger.info(bam_fh)` or
`msg = bam_fh  # format message however you like here
self.logger.info(msg)`

Reply:
I think this is just a leftover debug statement from implementation. Deleting it